### DISCUSSION TITLE
DIscord Buttons Support

### DISCUSSION THREAD

**Discussion by FeelsBadMan1** [Rapptz/discord.py#6967] [Q&A]
Hello, I saw others asking for Support with the new Buttons and i got a simmilar issure. 
I tried to use discord_buttons & discord components but none of these worked so i asked around on discord and they told me its only workig with the discord.py module. im not sure how the syntax is for using buttons within discord.py and the other issures where about 7 hours ago but within this time new files where added into discord.py module that also include buttons. however i get some issures related to these buttons here on pastebin what is a textuploading website i pasted 2 errors i get while using components and buttons. If anyone knows a solution, i would thank this person 1000 times it would help me a lot. However here is the pastebinurl with the errorlogs (expire after 1 week (might do the expiretime longer soon))
https://pastebin.com/mDn45k8D

**Comment by Rapptz** [comment]
This is for discord.py help, not unofficial 3rd party libraries like `discord_components` or `discord_buttons`. Buttons and components will be released in v2.0 of the library.

**Reply by TechStudent10** [reply]
k nice.

**Reply by JDJGInc** [reply]
Buttons and selects are built in now, other components aren't added quite yet.
